models-info/src/main/java/com/flipkart/fdp/ml/transformer/VectorAssemblerTransformer.java

```diff
@@ -20,27 +20,44 @@ public VectorAssemblerTransformer(final VectorAssemblerModelInfo modelInfo) {
     }
 
     private double[] predict (Object[] inputs) {
-        List<Double> output = new ArrayList<>();
+
+        int vectorSize = Stream.of(inputs)
```

Comment:
wanted to throw specific runtime exception messages for null and  non supported types
inputs.length will not work because it will non consider the length of inner arrays

e.g inputs = [0.1, [0.2,0.3]]
inputs.length = 2

where as we want to create output array of length 3